<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CoordSystemEditCtrl</class>
 <widget class="QWidget" name="CoordSystemEditCtrl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>544</width>
    <height>25</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CoordSystemEditCtrl</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_49">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../../../themes/green/green.qrc">:/images/view/coord.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>当前坐标系：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_ctrlpoint">
       <property name="text">
        <string>CGCS2000 1 3-degree Gauss Kruger CM 114E (EPSG:4547)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnEditCoord">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>编辑</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_24">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>37</width>
         <height>17</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../../themes/green/green.qrc"/>
  <include location="../themes/green/green.qrc"/>
 </resources>
 <connections/>
</ui>
